cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -906,7 +906,12 @@ impl UpgradeResult {
             for (prev_system, prev_package) in prev_packages_by_system {
                 // We must have the same packages before and after upgrading
                 let after_package = after_packages_by_system.remove(&prev_system).unwrap();
-                if prev_package.derivation() != after_package.derivation() {
+                // Store paths return None for the derivation,
```

Comment:
Did this ever report store paths as upgraded? After all `None == None` anyway so there shouldnt be a diff containing store paths already without manually verifying that the store paths are `Some`?